internal/engine/buildcontrol/live_update_build_and_deployer.go

```diff
@@ -50,15 +51,28 @@ func NewLiveUpdateBuildAndDeployer(dcu *containerupdate.DockerUpdater,
 }
 
 // Info needed to perform a live update
-type liveUpdInfo struct {
-	iTarget      model.ImageTarget
-	containers   []liveupdates.Container
-	changedFiles []build.PathMapping
-	runs         []model.Run
-	hotReload    bool
+type LiveUpdateInput struct {
+	ID     model.TargetID
+	Name   string
+	Filter model.PathMatcher
```

Comment:
Now everything here is part of the API model except the Filter! (which I'm talking to @milas about on a separate thread)